.travis.yml

```diff
@@ -35,7 +31,9 @@ before_install:
 install:
   # Customise the testing environment
   # ---------------------------------
+  - conda install --channel ioos pytest
   - conda install --channel ioos --file requirements.txt
-  - conda install --channel ioos nco pyncml pytest pynco
+  - conda install --channel ioos pyncml pynco
```

Comment:
What would you think of adding

```
conda config --add channels conda-forge
conda config --add channels ioos
```

to the `before_install` section and dropping the `--channel` here?